subsys/bluetooth/controller/flash/soc_flash_nrf_ticker.c

```diff
@@ -136,7 +136,7 @@ static void time_slot_delay(uint32_t ticks_at_expire, uint32_t ticks_delay,
 		_ticker_sync_context.result = 0;
 
 		/* Abort flash prepare ticker, from ULL_HIGH context */
-		ret = ticker_stop(instance_index, 1U, ticker_id,
+		ret = ticker_stop(instance_index, 1U, (ticker_id + 1),
```

Comment:
Correct ticker that was successfully started is ‘ticker_id’, hence ‘ticker_id’ is to be stopped not ‘ticker + 1’, that failed to start. Current implementation is correct.

Reply:
Thanks, I See.